extensions/amp-analytics/0.1/vendors.js

```diff
@@ -644,6 +644,76 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'linkpulse': {
+    'vars': {
+      'id': '',
+      'pageUrl': 'CANONICAL_URL',
+      'title': 'TITLE',
+      'section': '',
+      'channel': 'amp',
+      'type': '',
+      'host': 'pp.lp4.io',
```

Comment:
why is host a variable? Do you expect publishers to change it? Same for some of the other variables above.